.github/workflows/downstream-native-image.yaml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+# Keeping this file separate as the dependencies check would use more
+# repositories than needed this downstream check for GraalVM native image
+# compilation.
+name: downstream
+jobs:
+  # GraalVM job ensures the compatibility of GraaVM version
+  graalvm:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        graalvm: [22.3.0]
+        java: [11, 17]
+        repo:
+        # GAPIC library that doesn't use a real GCP project in integration tests
+        - orgpolicy
```

Comment:
@mpeddada1 This has been using the old orgpolicy repository as the canary. If you're good, I think it's better to replace this with the showcase module in this repository. With that, it won't need the pom manipulation in downstream-client-library-check.sh because showcase does not rely on the shared dependencies BOM.

Reply:
@suztomo Adding testing for the showcase module is a good idea! I'm curious, how big of a change would it be to replace the canary library with it?

Reply:
```
cd showcase
mvn test -P native
```

Reply:
Unit tests are disabled for showcase due to issues with the generated REST unit tests. If they're needed for native testing, we will need to explicitly ignore them.

Reply:
Unit tests are excluded from native test already.

Reply:
+1 Native image tests opt-in classes based on the `IT*` and `*ClientTest` naming conventions.